hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/airdrops/HRCTokenRejectTest.java

```diff
@@ -41,6 +42,7 @@
 import org.junit.jupiter.api.Tag;
 
 @Tag(SMART_CONTRACT)
+@OrderedInIsolation
```

Comment:
is there some unwanted interaction that required this?

Reply:
Yes, the whole suite was timing out when running embedded for some reason and Ordering solved that.